app/routes/profile.py

```diff
@@ -0,0 +1,24 @@
+from flask import request , jsonify , Blueprint ,current_app
+import jwt
+from app.model import User,Role
+from app import db
+
+profiles=Blueprint("/profile",__name__)
+@profiles.route('/userprofile', methods =['GET'])
+def  userprofile():
```

Comment:
This should be inside user.py,
Since there will be more functionalities for user, Not only get profile